app/src/main/AndroidManifest.xml

```diff
@@ -59,6 +59,7 @@
 
         <activity android:name=".activites.create.RescueCodeShowActivity"
             android:label="@string/title_rescuecode_show"
+            android:windowSoftInputMode="stateHidden"
```

Comment:
I wonder if we don't want the stateAlwaysHidden instead for this activity. I'm not sure we could navigate back to it today but maybe that would be an issue in the future.

"stateAlwaysHidden" | The soft keyboard is always hidden when the activity's main window has input focus.

Reply:
I agree. I'll change it to `stateAlwaysHidden`.
Thanks for taking time to review! :)

Reply:
Hi @alexhauser 

Thank you for giving your time to this repository. And helping me with code review and testing on my pull requests.

The quality of this project is important and if we can help each other so we find issues early then that's a win-win for everyone.

Best regards
Daniel